src/fal/cli/fal_runner.py

```diff
@@ -13,24 +13,21 @@
 def create_fal_dbt(
     args: argparse.Namespace, generated_models: Dict[str, Path] = {}
 ) -> FalDbt:
-    real_project_dir = os.path.realpath(os.path.normpath(args.project_dir))
-    real_profiles_dir = None
     env_profiles_dir = os.getenv("DBT_PROFILES_DIR")
+
+    profiles_dir = DEFAULT_PROFILES_DIR
     if args.profiles_dir is not None:
-        real_profiles_dir = os.path.realpath(os.path.normpath(args.profiles_dir))
+        profiles_dir = args.profiles_dir
     elif env_profiles_dir:
-        real_profiles_dir = os.path.realpath(os.path.normpath(env_profiles_dir))
-    else:
-        real_profiles_dir = DEFAULT_PROFILES_DIR
+        profiles_dir = env_profiles_dir
 
+    real_state = None
     if hasattr(args, "state") and args.state is not None:
-        real_state = Path(os.path.realpath(os.path.normpath(args.state)))
-    else:
-        real_state = None
+        real_state = args.state
 
     return FalDbt(
```

Comment:
Can we test creating FalDbt with both absolute and ~ paths?

Reply:
It can be a problem passing a `~` directory in tests, it is outside of the project and we don't know where it is cloned for everyone. I think we shouldn't. What other way can we test this?